scripts/load_raw_to_bq.py

```
@@ -407,6 +407,22 @@
 # when the DB is empty or corrupted.
 MIN_ROWS_TO_UPLOAD = 1000
 
+# Per-table override for the minimum-row threshold.
+# Some tables are sparse by design (event-targeted backfill, rare phenomena)
+# and would never reach the global 1000-row floor in their bootstrap phase.
+# These tables still need the threshold > 0 so a corrupt-empty SQLite cannot
+# WRITE_TRUNCATE the BQ data to zero — just at a lower level appropriate for
+# the data's natural density.
+TABLE_MIN_ROWS_OVERRIDE = {
+    # F-net broadband waveform: targeted around M6+ earthquakes, ~14 active
+    # stations × ~3 components × a handful of recent dates → low hundreds
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)